editor/script_editor_debugger.cpp

```diff
@@ -719,6 +719,9 @@ void ScriptEditorDebugger::_parse_message(const String &p_msg, const Array &p_da
 		packet_peer_stream->put_var(oe.callstack);
 		*/
 
+		// Show newest errors - first
+		error_list->invert_list();
```

Comment:
I think calling this repeatedly is going to get you an error list in the shape of:

newest
oldest
second

rather than
newest
second
oldest